03-nlp/languages.http

```diff
@@ -0,0 +1,255 @@
+Reference: https://learn.microsoft.com/en-gb/rest/api/language/text-analysis-runtime/analyze-text?view=rest-language-2023-04-01&tabs=HTTP
+
+### Language Detection
+POST {{ language endpoint }} /language/:analyze-text?api-version=2023-04-01 HTTP/1.1
+Content-Type: application/json
+Ocp-Apim-Subscription-Key: {{ Language-Key }}
+
+{
+  "kind": "LanguageDetection",
+  "parameters": {
+    "modelVersion": "latest"
+  },
+  "analysisInput": {
+    "documents": [
+      {
+        "id": "1",
+        "text": "Hello world"
+      },
+      {
+        "id": "2",
+        "text": "Bonjour tout le monde"
+      },
+      {
+        "id": "3",
+        "text": "Hola mundo"
+      },
+      {
+        "id": "4",
+        "text": "Tumhara naam kya hai?"
+      }
+    ]
+  }
+}
+
+### Entity Linking
+POST {{ language endpoint }} /language/:analyze-text?api-version=2023-04-01 HTTP/1.1
+Content-Type: application/json
+Ocp-Apim-Subscription-Key: {{ Language-Key }}
+
+{
+  "kind": "EntityLinking",
+  "parameters": {
+    "modelVersion": "latest"
+  },
+  "analysisInput": {
+    "documents": [
+      {
+        "id": "1",
+        "language": "en",
+        "text": "Microsoft was founded by Bill Gates and Paul Allen."
+      },
+      {
+        "id": "2",
+        "language": "en",
+        "text": "Pike place market is my favorite Seattle attraction."
+      }
+    ]
+  }
+}
+
+### Sentiment Analysis
+POST {{ language endpoint }}/language/:analyze-text?api-version=2023-04-01 HTTP/1.1
```

Comment:
- Comments in `.http` files should only have one `#`. The three `###` are how the REST Client extension separates REST requests. One `#` is for comments.
- Please use camelCase for the variables, so the language endpoint should be `languageEndpoint` which is consistent to the others we have in the repo.